src/Microsoft.DotNet.TemplateLocator/TemplateLocator.cs

```diff
@@ -5,68 +5,106 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using NuGet.Common;
-using NuGet.Protocol;
+using Microsoft.DotNet.DotNetSdkResolver;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
 
 namespace Microsoft.DotNet.TemplateLocator
 {
     public sealed class TemplateLocator
     {
-        private DirectoryInfo _dotnetSdkTemplatesLocation;
+        private readonly Lazy<NETCoreSdkResolver> _netCoreSdkResolver;
```

Comment:
minor: seeing how this is _readonly_ & gets instantiated in a .ctor, why is it a Lazy object?

Reply:
We don't need this initialized in a non VS scenario